thorlcr/graph/thgraph.hpp

```diff
@@ -632,6 +633,7 @@ class graph_decl CGraphBase : public CGraphStub, implements IEclGraphResults
     CGraphStubArrayCopy orderedChildGraphs;
     Owned<IGraphTempHandler> tmpHandler;
     AtomicShared<CFileSizeTracker> tempFileSizeTracker;
+    AtomicShared<IFileReadPropertiesUpdater> fileReadPropsUpdater;
```

Comment:
see other comment, but I think all should move to CMasterGraph, as CGraphBase (and workers) don't/can't have access to IDistributedFile's, dali etc.